markdown/developers/training_guide.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Training material guide
+subtitle: A guide to writing instructions for training material.
+---
+
+# Contributing a training guide to nf-core
+
+Thank you for helping us write a guide to using an aspect of nf-core.
+
+## Links
+
+- [Questions](#questions)
+- [Objectives](#objectives)
+- [Glossary](#glossary)
+- [Before I begin](#before-i-begin)
+- [How to](#how-to)
+  - [Bytesize](#bytesize-talk)
+  - [Walkthrough](#walkthrough)
+- [Key points](#key-points)
+- [References](#references)
+
+## Questions
+
+- How can I contribute a training guide to nf-core?
+- Where do I put the guide?
+- How do I structure content?
+
+## Objectives
+
+- Learn how to structure an nf-core guide.
+- Know where to submit the pull request.
+
+## Glossary
+
+<details>
+    <summary>Click here to expand the glossary of potentially unfamiliar terms</summary>
+    - **Learning Objective:** A predictive statement describing the specific outcomes that a training session is intended to achieve.
+    - **Key point:** An important take home message.
+    - **Cognitive load:** This relates to the amount of information that working memory can hold at one time.
+    - **Fork a repository:** This makes a copy of the repository to your personal GitHub workspace.
+    - **Make a branch:** A branch is another version of the files in your repository, including a history of how files were changed from the start of the branch to the current state of files.
+</details>
+
+## Before I begin
+
+A checklist of what you need to begin.
+
+- Ask on #training about the guide you'd like to contribute.
+- Make an issue on [nf-core website issues](https://github.com/nf-core/nf-co.re/issues) describing what you want to do, and assign yourself.
+- GitHub account.
+
+## How to
+
+### Bytesize talk
+
+There is no bytesize talk at present, but if you'd like present one, please make a suggestion on the [Slack #bytesize channel](https://nfcore.slack.com/channels/bytesize).
+
+### Walkthrough
+
+#### Use this as a template.
+
+Use this file `https://github.com/nf-core/nf-co.re/markdown/developers/training_guide.md` as a template to guide you.
```

Comment:
There's currently no mention of adding the new docs to the website navigation. However, I have been contemplating changing how this works anyway as it's a faff (currently need to do it manually - would be better done based on the discovered markdown files, maybe with some YAML front matter).

So technically it's missing from the docs here but it's very likely to change soon.

Reply:
Stuff still works without the link, it's just that you need to enter the URL manually.

Reply:
I decided on this because a navigation rework should be part of the group 4 task. People need somewhere to put the files until that is complete. 
But in case no decision on how to rework the navigation is made, I guess instructions need to be added on how to manually link it. 
Is this directory suitable to dump the training matter, or should it be put somewhere else?